src/components/post/postList/DeletePost.js

```diff
@@ -0,0 +1,82 @@
+import React, { Fragment, useState } from 'react';
+
+// Components
+import CustomButton from '../../../util/CustomButton';
+
+// MUI
+import withStyles from '@material-ui/core/styles/withStyles';
+import style from '../../../style/style';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogActions from '@material-ui/core/DialogActions';
+
+// Icons
+import DeleteOutline from '@material-ui/icons/DeleteOutline';
+
+// Redux
+import { connect } from 'react-redux';
+import { deleteUserPost } from '../../../redux/actions/dataActions';
+
+const DeletePost = ({
+  classes,
+  postId,
+  userHandle,
+  isAuthenticated,
+  handle,
+}) => {
+  const [openState, setOpenState] = useState({
+    open: false,
+  });
+  const { open } = openState;
+  const handleOpen = () => {
+    setOpenState({ ...openState, open: true });
```

Comment:
Technically you _could_ skip the {} on these and just return the executed function:
`()=> setOpenState(blabla);`
It saves a line or two but is more style choice than anything.

Reply:
fixed